src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -227,6 +231,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 Return False
             End Function
+
+            Public Overrides ReadOnly Property OverloadResolutionPriority As Integer
+                Get
+                    Return m_Method.OriginalDefinition.OverloadResolutionPriority
```

Comment:
This is probably wrong; we're using this method for filtering whether overload resolution priority is present on any methods, but `m_Method` isn't the method that has the priority we care about. We should only be examining the least overridden method for overload resolution priority. Please make sure we have tests covering overridden methods where the overridable method has priority, as well as methods where invalid IL resets the overridding method's priority to 0.

Reply:
This doesn't affect correctness and is suitable for a cheap check, overrides return 0, given how symbols implement the property

Reply:
Given the way VB name lookup works, the least derived forms are also among the candidates. Therefore we get to them as well

Reply:
Thanks for the clarification, that wasn't apparent to me at first read.